src/index.js

```diff
@@ -38,7 +38,7 @@ const _onFsPressForwardCallPress_PressabilityAst = babylon.parseExpression(_onFs
 
 // this is the code that we will generate for Touchable
 const _onFsPressForwardCode = `_onFsPressForward = function(isLongPress) {
-  if (UIManager.onFsPressForward == null) {
+  if (UIManager == null || UIManager.onFsPressForward == null) {
```

Comment:
Same:

```suggestion
  if (!UIManager || !UIManager.onFsPressForward) {
```

We could shorten it further with option chaining (`?.`), but that would break IE 11 and other old browsers.